From: Jeremy Bicha <jeremy.bicha@canonical.com>
Date: Wed, 10 May 2023 16:59:33 -0400
Subject: docs: Switch shebang to bash since popd and pushd are bashisms

popd and pushed are not implemented in all shells

https://rgeissert.blogspot.com/2013/02/a-bashism-week-pushing-and-poping.html
(cherry picked from commit 1981c7b5b69f2ba607e42d0720802faaf251644f)

Forwarded: https://gitlab.gnome.org/GNOME/tracker/-/merge_requests/650
---
 docs/reference/libtracker-sparql/dist-docs.sh        | 2 +-
 docs/reference/libtracker-sparql/generate-devhelp.sh | 2 +-
 docs/reference/libtracker-sparql/generate-svgs.sh    | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/docs/reference/libtracker-sparql/dist-docs.sh b/docs/reference/libtracker-sparql/dist-docs.sh
index 5511d54..926d6d9 100644
--- a/docs/reference/libtracker-sparql/dist-docs.sh
+++ b/docs/reference/libtracker-sparql/dist-docs.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 pushd $MESON_BUILD_ROOT
 
 # Ensure the build tree is compiled, we need generated files
diff --git a/docs/reference/libtracker-sparql/generate-devhelp.sh b/docs/reference/libtracker-sparql/generate-devhelp.sh
index 3458145..4a4ac32 100755
--- a/docs/reference/libtracker-sparql/generate-devhelp.sh
+++ b/docs/reference/libtracker-sparql/generate-devhelp.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 docs_name=$1
 docs_path=$2
diff --git a/docs/reference/libtracker-sparql/generate-svgs.sh b/docs/reference/libtracker-sparql/generate-svgs.sh
index c1e15c1..9d3faf4 100644
--- a/docs/reference/libtracker-sparql/generate-svgs.sh
+++ b/docs/reference/libtracker-sparql/generate-svgs.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 command=$1
 docs_path=$2
 
